core/assets/bundles/bundle.properties

```diff
@@ -570,6 +570,7 @@ item.lead.description = A basic starter material. Used extensively in electronic
 item.coal.name = Coal
 item.coal.description = A common and readily available fuel.
 item.graphite.name = Graphite
+item.graphite.description = Mineralized carbon, used as ammunition as well as electrical insulation.
```

Comment:
Should be "used *for* ammunition"; don't use "as" three times in a sentence.

Reply:
oof
ill fix it

Reply:
@anuken i might start doing this more often
i hope ill get better eventually